app/src/components/main/paginationToolbar/paginationToolbar.jsx

```diff
@@ -17,13 +17,18 @@ export const PaginationToolbar = ({
   onChangePageSize,
 }) => (
   <div className={cx('pagination-toolbar')}>
-    {itemCount && <ItemCounter activePage={activePage} pageSize={pageSize} itemCount={itemCount} />}
+    {itemCount === 0 && <NoItemMessage />}
```

Comment:
The purpose of the `PaginationToolbar` component is to show and control information about pagination. It shouldn't care about the content itself. That's why it's not the best place to show any messages about missing content.
Please move usage of `NoItemMessage` out of this component.

Reply:
fixed